pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -991,7 +994,11 @@ public void loadNamespaceTopics(NamespaceBundle bundle) {
             for (String topic : getNamespaceService().getListOfPersistentTopics(nsName).join()) {
                 try {
                     TopicName topicName = TopicName.get(topic);
-                    if (bundle.includes(topicName)) {
+                    if (bundle.includes(topicName)
+                            && !topic.startsWith(TopicName.TRANSACTION_COORDINATOR_ASSIGN.toString())
```

Comment:
does it make sense to create a static method with this list of tests ?

`private static boolean isTransactionSystemTopic(TopicName topicName) ...`